docker-compose.yml

```diff
@@ -26,8 +26,8 @@ x-environment:
     GOVCMS_IMAGE_VERSION: ${GOVCMS_IMAGE_VERSION:-latest}
     DEV_MODE: ${DEV_MODE:-false}
     X_FRAME_OPTIONS: ${X_FRAME_OPTIONS:-SameOrigin}
-    DRUPAL_SHIELD_USER: ${DRUPAL_SHIELD_USER:-govcms}
-    DRUPAL_SHIELD_PASS: ${DRUPAL_SHIELD_PASS:-smcvog}
+    DRUPAL_SHIELD_USER: ${DRUPAL_SHIELD_USER}
```

Comment:
This will give you `WARNING: The DRUPAL_SHIELD_USER variable is not set. Defaulting to a blank string.` warning if `.env` is not available.

Could you please change it to `DRUPAL_SHIELD_USER: ${DRUPAL_SHIELD_USER:-}`